source/manual/2nd-line-drills.html.md

```diff
@@ -81,9 +81,47 @@ See "[If Slack is unavailable](https://docs.google.com/document/d/144y8c2Ly-kG3J
 In preparation for a spike in traffic, you can increase the number of unicorn workers for an app.
 See [established connections exceeded](/manual/alerts/established-connections-exceed.html) for details.
 
-Pick an application and drill scaling up the number of workers - see [example](https://github.com/alphagov/govuk-puppet/pull/11194).
+Pick an application and drill scaling up the number of workers.
 
-You can create a branch of `govuk-puppet` and deploy that branch to Integration to see the unicorn worker change take effect. Delete the branch and re-deploy the latest release of Puppet when you're done.
+### Check the number of unicorn workers currently running
```

Comment:
We're trying to keep the `2nd-line-drills.html.md` file as succinct as possible (basically linking out to separate instructions in each case), so let's move all the content from this line onwards.

I can't find a good home for it, so I'd suggest a new file, e.g. `manual/scale-unicorn-workers.html.md`, and put the content in there.

You can also then link to that new file from https://docs.publishing.service.gov.uk/manual/alerts/established-connections-exceed.html, improving the alert documentation 👍

Reply:
Thanks, I've created the new document and linked to it from https://docs.publishing.service.gov.uk/manual/alerts/established-connections-exceed.html 👍